config/locales/curation_concerns.en.yml

```diff
@@ -0,0 +1,117 @@
+en:
+  curation_concerns:
```

Comment:
You should not copy this whole file here.  You only need to put the keys you want to change in your `config/locales/en.yml` if you copy the whole file, upgrading curation_concerns will be more difficult.